change/@fluentui-babel-make-styles-e24db7df-1613-42dd-aa1a-a4ddb5b85b83.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "fix: handle computed keys on objects",
+  "packageName": "@fluentui/babel-make-styles",
+  "email": "olfedias@microsoft.com",
+  "dependentChangeType": "patch"
```

```suggestion
  "dependentChangeType": "none"
```

Dev dependency, no need to bump all packages